# Advantages of CEMiTool for gene co-expression analysis of RNA-seq data
This repository contains codes that I used in my recent publication in Computers in Biology and Medicine (Cheng et al., 2020)

[Link to publication](https://www.sciencedirect.com/science/article/abs/pii/S0010482520303061)

[Click here to see codes](https://github.com/Cheng-CW/Advantages-of-CEMiTool-for-gene-co-expression-analysis-of-RNA-seq-data/blob/master/codes)

# Citations
```
@article{CHENG2020103975,
title = "Advantages of CEMiTool for gene co-expression analysis of RNA-seq data",
journal = "Computers in Biology and Medicine",
volume = "125",
pages = "103975",
year = "2020",
issn = "0010-4825",
doi = "https://doi.org/10.1016/j.compbiomed.2020.103975",
url = "http://www.sciencedirect.com/science/article/pii/S0010482520303061",
author = "Chew Weng Cheng and David J. Beech and Stephen B. Wheatcroft",
keywords = "RNA-seq, Co-expression, Ischemic cardiomyopathy, Chronic obstructive pulmonary Disease, WGCNA, CEMiTool, Coseq",
abstract = "Gene co-expression analysis is widely applied to transcriptomics data to associate clusters of genes with biological functions or identify therapeutic targets in diseases. Recently, the emergence of high-throughput technologies for gene expression analyses allows researchers to establish connections through gene co-expression analysis to identify clinical disease markers. However, gene co-expression analysis is complex and may be a daunting task. Here, we evaluate three co-expression analysis packages (WGCNA, CEMiTool, and coseq) using published RNA-seq datasets derived from ischemic cardiomyopathy and chronic obstructive pulmonary disease. Results show that the packages produced consensus co-expression clusters using default parameters. CEMiTool package outperformed the other two packages and required less computational resource and bioinformatics experience. This evaluation provides a basis on which data analysts can select bioinformatics tools for gene co-expression analysis."
}
```
